### DISCUSSION TITLE
Idea for IDE-Feature: Jump to file in Project-Explorer

### DISCUSSION THREAD

**Discussion by bernd5** [dotnet/roslyn#52046] [Ideas] [answered]
For large solutions it would be nice to have a "Jump/Navigate to file in Project-Explorer", similar to "Open containing folder" in the file-tab context menu.
This would be really great.

**Comment by CyrusNajmabadi** [comment, marked as answer]
This exists.  You can either enable:

![image](https://user-images.githubusercontent.com/4564579/112033192-68d24580-8afa-11eb-984a-030d15c5a828.png)

Or, if htat is off, you just click on this button:

![image](https://user-images.githubusercontent.com/4564579/112033242-75ef3480-8afa-11eb-9225-fae0aa233992.png)

**Reply by bernd5** [reply]
Thank you very much, I never recognized it 😀
